datasource/src/main/resources/liquibase/changelog/changelog-0.1.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <include file="../properties.xml" relativeToChangelogFile="true"/>
+
+    <changeSet id="create_user_table" author="alitkovetc">
+        <createTable tableName="user">
+            <column name="user_id" type="${uuid.type}"/>
+            <column name="email" type="${text.type}"/>
```

Comment:
Email/password/user_name should have a reasonable length limit, so use varchar instead of text.

Reply:
Done

Reply:
@timaliberdov why? What is considerable length? Text is as good in performance as varchar

Here is my email:
ireallywanttogetthismessagefromyousomuchtimur@iliaisherenow.33mail.com
It really works and maybe I really want to use pipeline service...

Reply:
@ilyaonishenko, that's true for PostgreSQL. As for MySQL: [TEXT and BLOB fields are not stored in-row](https://dev.mysql.com/doc/refman/5.6/en/memory-storage-engine.html#memory-storage-engine-characteristics-of-memory-tables) — they require a separate lookup (and a potential disk read) if their column is included in a SELECT clause. Additionally, the inclusion of a TEXT or BLOB column in any sort will force the sort to use a disk-based temporary table.

Reply:
If you want to use just PostgreSQL then it's fine.

Reply:
The probability of database change is almost the same as for user with email length > 70 :)